doc/development/deprecations.rst

```diff
@@ -17,13 +29,13 @@ Pending deprecations
   ``QuantumTape`` instead.
 
   - Still accessible in v0.28, v0.29
-  - Will be removed in v0.30
+  - Will be removed in v0.31
 
 * The ``grouping`` module is deprecated. The functionality has been moved and
   reorganized in the new ``pauli`` module under ``pauli/utils.py`` or ``pauli/grouping/``.
 
   - Still accessible in v0.27, v0.28, v0.29
-  - Will be removed in v0.30
+  - Will be removed in v0.31
```

Comment:
This is unrelated I suppose? Is this a surprise/did we miss to realize what we announced here? (same above).
Also, should the "still accessible" lines include 0.30, then?

Reply:
I just noticed it while I was adding this to the Changelog. As of now it hasn't been removed 😢, since this is the last day of feature freeze I doubt it will get removed this time. I just updated it here, I can add v0.30. Really this should also be changed in the RC-branch 👍🏼

Reply:
Are you taking care of that? :)

Reply:
Made a separate PR for this 👍🏼